Packs/GreyNoise/Integrations/GreyNoise/GreyNoise.py

```diff
@@ -1106,6 +1108,150 @@ def context_command(client: Client, args: dict, reliability: str) -> CommandResu
     )
 
 
+@exception_handler
+@logger
+def cve_command(client: Client, args: dict, reliability: str) -> CommandResults:
+    """
+    Returns information about CVE.
+
+    :type client: ``Client``
+    :param client: client object
+
+    :type args: ``dict``
+    :param args: All command arguments, usually passed from ``demisto.args()``.
+    :return: A ``CommandResults`` object that is then passed to ``return_results``,
+           that contains the IP information.
+    :rtype: ``CommandResults``
+
+    :type reliability: ``String``
+    :param reliability: string
+    """
+    cve_arg = args.get("cve", "")
+    cvss = 0
+    description = ""
+    published = ""
+    modified = ""
+    response = client.cve(cve_arg)
+    cve_raw_response = copy.deepcopy(response)
+    response = remove_empty_elements(response)
+    if response.get("id"):
+        cvss = response["details"].get("cve_cvss_score", "N\\A")
```

Comment:
two questions:
1. Does the field "details" exist in this case? (since we access with [] and not get.())
2.  "N\\A" is it necessary? if the **cve_cvss_score** does not exist we gets **None** any way.